### ISSUE TITLE
Editor crashes after editing textarea if provider uses transformCode

### ISSUE THREAD

**Issue by NoemiRozpara** [FormidableLabs/react-live#126]
Hi guys, 

today I noticed issue with transformCode prop. In my case I'm showing only button in textarea to make it editable, but want to wrap it with some nice looking component. I used transformCode prop to wrap the button before render. It renders nice, but it crashes after first text edition. The error says:

> Invariant Violation: Unexpected text node: [object Object]. A text node cannot be a child

When transformCode function logs received code to the console, it looks like it is receiving synthetic event at first, then throws error, then receives code as string and logs it again. My problem was solved by checking if received code is a string and to return wrapped code only if it is. But maybe this could be fixed? 

Code: 
```
const code = `<Button> 
  Sign me in! 
</Button>`;

const transformCode = code => {
  console.log(code);
  // if (typeof code === 'string') { // this fixes bug
  return `<View>
          <Title> Some title without edition </Title>
         ${code}
        </View>`;
  //  }
};

const scope = {
  View,
  Button,
  Title
};

const Editor = () => (
  <View>
    <LiveProvider code={code} scope={scope} transformCode={transformCode}>
      <LiveEditor style={{ borderStyle: 'solid' }} />
      <LiveError style={{ borderStyle: 'solid' }} />
      <LivePreview style={{ borderStyle: 'solid' }} />
    </LiveProvider>
  </View>
);
```
Screens: 
Editor right after render
<img width="1027" alt="Zrzut ekranu 2019-03-27 o 14 20 27" src="https://user-images.githubusercontent.com/27765703/55079778-2c9bb400-509d-11e9-9170-bff5d465c881.png">

One char edited
<img width="1210" alt="Zrzut ekranu 2019-03-27 o 14 21 37" src="https://user-images.githubusercontent.com/27765703/55079859-4fc66380-509d-11e9-8dff-27c62d00c8d8.png">

Console content
<img width="782" alt="Zrzut ekranu 2019-03-27 o 14 21 19" src="https://user-images.githubusercontent.com/27765703/55079868-5523ae00-509d-11e9-93bc-62f09b3cb4bb.png">

**Comment by satya164** [comment]
cc @sofiapoh

**Comment by viczam** [comment]
I believe it's related to #127

**Comment by sofiapoh** [comment]
I've just released a patch with #132 which should fix this, let me know if there are still some problems!
